stores/sql/mysql/migrations/main/migration_00027_contract_usability.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE contracts ADD COLUMN usability tinyint unsigned NOT NULL;
+CREATE INDEX `idx_contracts_usability` ON contracts (`usability`);
+UPDATE contracts SET usability = 1 WHERE 1=1;
```

Comment:
Why not set all contracts to "good" if you are going to have to do that eventually anyway to avoid sudden churn?

Reply:
Planned to F/U with that when integrating it into the contractor